packages/aws-cdk-lib/aws-rds/README.md

```diff
@@ -938,6 +938,7 @@ Data in S3 buckets can be imported to and exported from certain database engines
 functionality, set the `s3ImportBuckets` and `s3ExportBuckets` properties for import and export respectively. When
 configured, the CDK automatically creates and configures IAM roles as required.
 Additionally, the `s3ImportRole` and `s3ExportRole` properties can be used to set this role directly.
+Note that when using  `s3ImportRole` and/or `s3ExportRole` with AuroraPostgres, you need to enable the respectives feature when declaring the DatabaseClusterEngine.
```

Comment:
What is the 'respectives' feature? Having trouble finding out more information about it.

Reply:
Feel free to rephrase - good to have an external view

Reply:
Is it accurate to say this?
> To use s3ImportRole and s3ExportRole with Aurora PostgreSQL, you must also configure the S3 import and export features when creating the DatabaseClusterEngine.

Reply:
I think enable would be more appropriate - as it is turning a boolean flag to true